azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/__init__.py

```diff
@@ -23,31 +32,51 @@ def configure_azure_monitor(**kwargs):
     """
 
     configurations = get_configurations(**kwargs)
-    disable_tracing = configurations.get("disable_tracing", False)
+    connection_string = configurations.get("connection_string", False)
     service_name = configurations.get("service_name", "")
     service_namespace = configurations.get("service_namespace", "")
     service_instance_id = configurations.get("service_instance_id", "")
+    disable_logging = configurations.get("disable_logging", False)
+    logging_level = configurations.get("logging_level", NOTSET)
```

Comment:
Default logging level is WARNING for python logging library so we should probably stick to that.

Reply:
Will double check, but I believe the handler is set to NOTSET by default while the loggers themselves are set to WARN.

Reply:
LoggingHandler defaults to NOTSET: https://github.com/open-telemetry/opentelemetry-python/blob/release/v1.14.x-0.35bx/opentelemetry-sdk/src/opentelemetry/sdk/_logs/__init__.py#L317

Reply:
Same for sitecustomize flow: https://github.com/open-telemetry/opentelemetry-python/blob/release/v1.14.x-0.35bx/opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py#L224